Assets/Scripts/PauseController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class PauseController : MonoBehaviour {
+
+    public Transform pauseMenu;
+    public Transform mecha;
+
+    public Slider slider;
```

Comment:
If you force the initialization of an attribute in `Start()`, there is no point in making it `public`